lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenHoverParticipant.java

```diff
@@ -326,17 +343,23 @@ public static String getMavenPropertyInRequest(IPositionRequest request) {
 		return null;
 	}
 
-	private Hover collectProperty(IPositionRequest request, String property) {
+	private Hover collectProperty(IHoverRequest request, String property) {
+		boolean supportsMarkdown = request.canSupportMarkupKind(MarkupKind.MARKDOWN);
 		DOMDocument doc = request.getXMLDocument();
 		MavenProject project = cache.getLastSuccessfulMavenProject(doc);
 		if (project != null) {
 			Map<String, String> allProps = getMavenProjectProperties(project);
+			UnaryOperator<String> toBold = message -> supportsMarkdown ? MarkdownUtils.toBold(message) : message;
```

Comment:
This can probably be replaced by `UnaryOperator<String> toBold = supportsMarkdown ? MarkdownUtils::toBold : UnaryOperator.identity()`